app/components/team_sidebar/add_team/team_list_item/team_list_item.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {DeviceEventEmitter, Text, View} from 'react-native';
+
+import {addUserToTeam} from '@actions/remote/team';
```

Comment:
yes in fact this adds the user to the team but it does not fetches channels for the team nor the posts of the default channel for tablets. Perhaps we need to augment the function